tests/by-util/test_pathchk.rs

```diff
@@ -19,8 +19,10 @@ fn test_default_mode() {
     // accept non-portable chars
     new_ucmd!().args(&["dir#/$file"]).succeeds().no_stdout();
 
-    // accept empty path
-    new_ucmd!().args(&[""]).succeeds().no_stdout();
+    // fail on empty path
+    new_ucmd!().args(&[""]).fails().no_stdout();
```

Comment:
please check the error message too :)

Reply:
I have change `no_stdout` to `stderr_only` :)